packages/hash/frontend/src/components/layout/PageSidebar/AccountEntityTypeList/SortActionsDropdown.tsx

```diff
@@ -0,0 +1,120 @@
+import { VFC } from "react";
+import { bindMenu, PopupState } from "material-ui-popup-state/hooks";
+import {
+  Box,
+  ListItemIcon,
+  ListItemText,
+  Menu,
+  MenuItem,
+  Tooltip,
+  Typography,
+} from "@mui/material";
+import {
+  faArrowDownAZ,
+  faArrowDownShortWide,
+  faArrowUpWideShort,
+  faArrowUpZA,
+  faQuestionCircle,
+  faWandSparkles,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "../../../icons";
+
+export type SortType = "asc" | "desc" | "recent" | "most_used" | "least_used";
+
+type SortActionsDropdownProps = {
+  setSortType: (sortType: SortType) => void;
+  activeSortType: SortType;
+  popupState: PopupState;
+};
+
+const menuItems: {
+  id: number;
+  title: string;
+  sortType: SortType;
+  icon: IconDefinition;
+}[] = [
+  {
+    id: 1,
+    title: "Alphabetical",
+    icon: faArrowDownAZ,
+    sortType: "asc",
+  },
+  {
+    id: 2,
+    title: "Reverse Alphabetical",
+    icon: faArrowUpZA,
+    sortType: "desc",
+  },
+  {
+    id: 3,
+    title: "Recently Updated",
+    icon: faWandSparkles, // @todo-mui get a free icon that matches the design closely
+    sortType: "recent",
+  },
+  {
+    id: 4,
+    title: "Most used",
+    icon: faArrowUpWideShort,
+    sortType: "most_used",
+  },
+  {
+    id: 5,
+    title: "Least used",
+    sortType: "least_used",
+    icon: faArrowDownShortWide,
+  },
+];
+
+export const SortActionsDropdown: VFC<SortActionsDropdownProps> = ({
+  setSortType,
+  activeSortType,
+  popupState,
+}) => {
+  return (
+    <Menu {...bindMenu(popupState)}>
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        alignItems="center"
+        pl={1.75}
+        pr={1.25}
+        py={1.25}
+      >
+        <Typography
+          variant="smallCaps"
+          sx={({ palette }) => ({
+            color: palette.gray[50],
+            fontWeight: 600,
+          })}
+        >
+          Sort By
+        </Typography>
+        <Tooltip title="Sort">
+          <FontAwesomeIcon
+            icon={faQuestionCircle}
```

Comment:
I'm only flagging this because in some places you place an explicit comment that you're using a different icon from the designs, but in others you're not so want to double check it's intentional.

Reply:
Yes I am aware. Sorry I should have dropped a todo comment to. A lot of the icons in the sidebar are pro icons and as such I had to use a free alternative for the time being. Once custom icons have been done by @MaggieAppleton I will replace. Created a [task](https://app.asana.com/0/1201095311341924/1202036166995552) to track this